lib/samples/query_feature_table/README.md

```diff
@@ -0,0 +1,35 @@
+# Query feature table
+
+Find features in a feature table which match an SQL query.
+
+![Image of query feature table](query_feature_table.png)
+
+## Use case
+
+Query expressions can be used in ArcGIS to select a subset of features from a feature table. This is most useful in large or complicated data sets. A possible use case might be on a feature table marking the location of street furniture through a city. A user may wish to query by a TYPE column to return "benches". In this sample, we query a U.S. state by STATE_NAME from a feature table containing all U.S. states.
```

Comment:
I think "through" was meant to say "throughout".

Reply:
I agree!